Wording changes in Updating Traffic Keys (#114)
tlswg/tls-key-update: draft-ietf-tls-extended-key-update.md

generation of application_traffic_secret is motivated by the desire
to include

* a secret derived from the negotiated key exchange, whether (EC)DHE, post-quantum KEM, or a PQ/T hybrid mechanism,
* a secret that allows the new key exchange to be cryptographically
bound to the previously established secret,
* a transcript hash that is updated after each Extended Key Update exchange by
hashing the previous transcript hash together with the current
ExtendedKeyUpdate(key_update_request) and
ExtendedKeyUpdate(key_update_response) messages. This binds any

> encapsulation ciphertexts used in PQ/T hybrid or post-quantum key exchanges

Reviewer: ```suggestion
encapsulation ciphertext and public key used in PQ/T hybrid or post-quantum key exchanges
```

to the IKM and cryptographically binds the newly derived secrets to the

> original handshake transcript, all prior EKU exchanges, and the current EKU

Reviewer: ```suggestion
original handshake transcript, all prior EKU exchanges, the current EKU
```

> exchange, and

Reviewer: ```suggestion
exchange, and the corresponding KEM public key and encapsulation ciphertext, and
```
